Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -1265,13 +1278,25 @@ void CGameListCtrl::AutomaticColumnWidth()
 			+ GetColumnWidth(COLUMN_EMULATION_STATE));
 
 		// We hide the Maker column if the window is too small
-		if (resizable > 400)
+		if (resizable > 425)
 		{
-			if (SConfig::GetInstance().m_showMakerColumn)
+			if (SConfig::GetInstance().m_showMakerColumn &&
+				SConfig::GetInstance().m_showFileNameColumn)
+			{
+				SetColumnWidth(COLUMN_TITLE, resizable / 3);
+				SetColumnWidth(COLUMN_MAKER, resizable / 3);
+				SetColumnWidth(COLUMN_FILENAME, resizable / 3);
+			}
```

Comment:
A space is missing between "else if" and the parenthesis. Idem for the next "else if" block